sdk/tests/conformance/resources/gl-object-get-calls.js

```diff
@@ -122,6 +121,11 @@ gl.bindTexture(gl.TEXTURE_2D, null);
 var framebuffer = gl.createFramebuffer();
 gl.bindFramebuffer(gl.FRAMEBUFFER, framebuffer);
 gl.framebufferTexture2D(gl.FRAMEBUFFER, gl.COLOR_ATTACHMENT0, gl.TEXTURE_2D, texture, 0);
+var colorAttachmentsNum = 1;
+if (contextVersion > 1) {
+  colorAttachmentsNum = gl.getParameter(gl.MAX_COLOR_ATTACHMENTS);
+  gl.framebufferTexture2D(gl.FRAMEBUFFER, gl.COLOR_ATTACHMENT0 + colorAttachmentsNum - 1, gl.TEXTURE_2D, texture, 0);
```

Comment:
The color attachment points in between are all empty. That's true. But The for loop doesn't test the color attachment points in between at all. So I think the code is OK. 

Well, I want to test more color attachment point(s) except the first one for WebGL 2, but testing all color attachment points seems to be unnecessary.  So this test choose the last one. So it binds texture to the first and the last color attachment points to the framebuffer, then test these two attachment points only.